SIT.Manager/Services/Install/InstallerService.cs

```diff
@@ -340,11 +340,16 @@ public Process CreatePatcherProcess(string patcherPath)
         };
         if (OperatingSystem.IsLinux())
         {
-            patcherProcess.StartInfo.FileName = _configService.Config.WineRunner;
+            // TODO: actually improve this (will probably be done after fixing launching problems)
+            LinuxConfig config = (LinuxConfig) _configService.Config;
+            string winePrefix = Path.GetFullPath(config.WinePrefix);
+            // Update the wine prefix and install any required components
+            UpdateWinePrefix(winePrefix);
```

Comment:
I'll add some frontend to this to show what is happening, but I won't ask the user whether they want to update the wine prefix as without proper components it will result in improper rendering.